app/page.tsx

```diff
@@ -1,5 +1,12 @@
 "use client";
-import { Grid } from "@chakra-ui/react";
+
+//LIBRARY IMPORTS
+import { Link } from "@chakra-ui/next-js";
```

Comment:
Imports for Link, Center, Box, Image and the divStyle const are no longer being used, so they can all be removed.